src/main/groovy/com/github/jrubygradle/JRubyPrepareJars.groovy

```diff
@@ -0,0 +1,55 @@
+package com.github.jrubygradle
+
+import com.github.jrubygradle.internal.JRubyExecUtils
+import org.gradle.api.DefaultTask
+import org.gradle.api.Project
+import org.gradle.api.file.FileCollection
+import org.gradle.api.tasks.InputFiles
+import org.gradle.api.tasks.OutputDirectory
+import org.gradle.api.tasks.TaskAction
+
+/**
+ * @author Schalk W. Cronjé
+ * @author R Tyler Croy
+ * @author Christian Meier
+ */
+class JRubyPrepareJars  extends DefaultTask {
+
+    /** Target directory for GEMs. Extracted GEMs should end up in {@code outputDir + "/jars"}
+     */
+    @OutputDirectory
+    File outputDir
+
+    /** Sets the output directory
+     *
+     * @param f Output directory
+     */
+    void outputDir(Object f) {
+        this.outputDir = project.file(f)
+    }
+
+    @TaskAction
+    void copy() {
```

Comment:
I don't see any reference to outputDir in `exec()`. Was your idea to pass it to the `into` method?